CONTRIBUTING.md

```diff
@@ -66,8 +65,7 @@ chances of your issue being dealt with quickly:
 - **Motivation for or Use Case** - explain what are you trying to do and why the current behavior
     is a bug for you
 - **Browsers and Operating System** - is this a problem with all browsers?
-- **Reproduce the Error** - provide a live example using our [Demo site][demo] or a
-    unambiguous set of steps
+- **Reproduce the Error** - Please fork our [starter app](https://github.com/GetTerminus/ui-stackblitz-starter) to replicate your issue and then share that link.
```

Comment:
One last item.. if you move this link down to the bottom with others, we'll only need to update it in one place